test/logger.test.js

```diff
@@ -2,6 +2,7 @@ import chai from 'chai';
 chai.should();
 
 import SafeLogger from '../src/main';
+import { AnotherError } from './utils/error';
```

Comment:
I think moving this error mock class into a new file makes it more difficult to understand these tests, as a developer I have to open another file to understand the data structure of the error.